src/pages/search/index.vue

```diff
@@ -28,7 +28,15 @@ const SearchIndex = defineComponent({
     const { canLoadMore, onLoadMore } = useLoadMore(props)
     return { canLoadMore, onLoadMore }
   },
-  head: {},
+  head: {
```

Comment:
The search page `title` is still using the `-` instead of `|`. Not sure how much that matters considering the search result pages are not meant to be indexed anyway but consistency is probably good.

Reply:
Fixed, ty!